docs/InputField.md

```diff
@@ -20,6 +20,52 @@ Make sure to provide an `id` prop to this component, otherwise you will run into
 ### `<Icon />`
 One `<Icon />` can go _before_, and one `<Icon />` can go _after_ the `<Input />` component. Anything else will result in a prop-type warning.
 
+## Error Handling
+All error handling should be performed by the parent component. An example of how one might accomplish this is below.
+
+```jsx
+<Box
+  width={'50%'}
+  bg='white'
+>
+  <InputField onChange={this.props.onChange}>
+    <Icon name='search' color='blue' size={18} />
+    <Input
```

Comment:
(n-b): Maybe better to just keep the props in the Input to what's relevant to an example for handling error. I would personally remove `id, onChange, color, icon, aria stuff, etc`. Let's keep it simple